templates/components/infocom.html.twig

```diff
@@ -324,7 +324,7 @@
          ) }}
       {% endif %}
 
-      {{ call_plugin_hook(constant('Glpi\\Plugin\\Hooks::INFOCOM'), item) }}
+      {{ call_plugin_hook(constant('Glpi\\Plugin\\Hooks::INFOCOM'), {'item': item}) }}
```

Comment:
Maybe hook result is not made to be displayed
```suggestion
      {% do call_plugin_hook(constant('Glpi\\Plugin\\Hooks::INFOCOM'), item) %}
```

Reply:
Yes i understand, but this hook permit to display more information after infocom form & your suggested change don't work on it.
Another solution is to delete hook in twig file & launch it at the end of showForItem function in src/Infocom.php

```
diff --git a/src/Infocom.php b/src/Infocom.php
index be86709c65..5eb537d86d 100644
--- a/src/Infocom.php
+++ b/src/Infocom.php
@@ -1043,6 +1043,7 @@ class Infocom extends CommonDBChild {
             'can_global_purge'  => Session::haveRight(self::$rightname, PURGE),
          ]);
       }
+      self::addPluginInfos($item);
    }
 
    static function addPluginInfos(CommonDBTM $item) {
```